repository/src/main/java/tech/ydb/yoj/repository/db/list/InMemoryQueries.java

```diff
@@ -205,6 +208,15 @@ private static boolean contains(@Nullable String input, @Nullable String substri
         return input.contains(substring);
     }
 
+    private static boolean iContains(@Nullable String input, @Nullable String substring) {
```

Comment:
We should use an explicit locale when calling String.toLowerCase() - Locale.ROOT is probably a sane choice.